cypress/e2e/Transcript/Access/BoardAndDoBIsRequiredToSomeBoards.feature

```diff
@@ -0,0 +1,16 @@
+Feature: The user is required to complete their profile to see the transcripts
+  Background:
+    Given I am a registered user
+    And I have selected that I have a medical board
+    And I did not provide the required <MedicalBoardInformation>
+
+  Scenario: user has empty board or date of birth details
+    Given an user did not fill the board details required by its board selection
```

Comment:
In general, we always write in present, but we have two options:

```
Given in past
When in present
Then in future
```

or

```
Given in present
When in present
Then in present
```

We have been using the last one for other tests